apps/website/prisma/schema.prisma

```diff
@@ -237,7 +237,7 @@ model Form {
   endAt              DateTime?
   outgoingWebhookUrl String?   @db.VarChar(720)
   showInLists        Boolean   @default(true)
-  config             String    @default("") @db.Text
+  config             String    @db.Text
```

Comment:
wasn't sure if the correct definition would be `String?` to indicate it is optional

Reply:
No, having it not optional should be the right thing. This way the config is required to be set on create.